functional-tests/fn_bridge_deposit_happy.py

```diff
@@ -5,11 +5,12 @@
 from strata_utils import deposit_request_transaction, drain_wallet
 
 from constants import DEFAULT_ROLLUP_PARAMS
-from utils import get_bridge_pubkey, get_logger
+from entry import TestStrata
+from utils import get_bridge_pubkey
 
 
 @flexitest.register
-class BridgeDepositHappyTest(flexitest.Test):
+class BridgeDepositHappyTest(TestStrata):
```

Comment:
It'd be fine to just directly say `testenv.StrataTest` here (and in all the other places) to avoid cluttering the imports.

Reply:
StrataTest gets the file get picked up as test file, so i had to resort to `StrataTester`

Reply:
That's annoying, I'll make an adjustment to the autoconfiguration later.

Reply:
`StrataTestBase` maybe would have been more appropriate but that's fine I'll just fix it later.